lib/msf/core/exploit/remote/dns/enumeration.rb

```diff
@@ -246,7 +246,7 @@ def dns_get_srv(domain)
           }
           report_note(
             type: srv_record,
-            data: srv_record_data
+            data: { :srv_record_data => srv_record_data }
```

Comment:
You'll have to double check that we're not wrapping values that are already objects; for instance here it looks like `srv_records_data` is already a hash, so we don't need to wrap it again